README.md

```diff
@@ -1,8 +1,33 @@
 # Universal Packaging Tool: MacPorts backend
 This is a [MacPorts](https://www.macports.org) backend for [upt](https://pypi.python.org/pypi/upt).
 
-#### Steps to use this tool out:
+The prototype was written by Mojca Miklavec (@mojca) and further developed as part of
+[Google Summer of Code 2019](https://summerofcode.withgoogle.com/archive/2019) by
+Karan Seth (@Korusuke), mentored by Cyril Roelandt (@Steap) and Renee Otten (@reneeotten).
 
-- Clone this repository and run ```python3 setup.py install``` to install macports backend
-- Run ```$ upt package -f pypi -b macports package_name```
-- Portfile output will be printed via stdout
+## Installation
+
+### Using MacPorts
+Follow [the instructions](https://www.macports.org/install.php) to install MacPorts on your Mac.
+
+Use the command below to install `upt`, the `upt-macports` backend, and all supported frontends:
+
+> sudo port install upt
+
+### Install from source
+The source can be obtained from [PyPI](https://pypi.org/project/upt/) or [framagit](https://framagit.org/upt/upt); several
+frontends are also available from these websites.
+
+To install the MacPorts backend, first clone the repository and follow the typical installation steps:
+```
+git clone https://github.com/macports/upt-macports.git
+cd upt-macports/
+python setup.py install
```

Comment:
Should we write `python3` instead as for systems with both versions installed `python` would result in `python2`, at least this is the behavior for my ubuntu installation.

Reply:
okay, I added a sentence to clarify what ```python``` should be